circuit/src/main/kotlin/com/slack/circuit/Circuit+Navigator.kt

```diff
@@ -113,32 +113,29 @@ internal fun <R : BackStack.Record> FactoryNavigator(
 }
 
 @Composable
-internal fun <R : BackStack.Record> BasicFactoryNavigator(
```

Comment:
We can look at making this more generic in the future if we want, but for now I think this makes sense. We wanna take a look at modifying the underlying backstack in the future anyway